### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mptre**

### CODE REVIEW

File: src/pick.c [hunk 1]
```diff
@@ -291,10 +292,10 @@ selected_choice(void)
...
 			if (cursor_position > 0) {
 				delete_between(
-				    query,
-				    query_length,
-				    cursor_position - 1,
-				    cursor_position);
+					       query,
+					       query_length,
+					       cursor_position - 1,
+					       cursor_position);
```
Comment (calleerlandsson) [inline src/pick.c]: I think these instances were correctly indented. I use `cinoptions=:0,t0,+4,(4` in Vim. What editor and configuration do you use?

**Comment by calleerlandsson** [comment]
Hey @mptre, thanks for cleaning up. I had one question about indentation. Other than that, this looks good!

File: src/pick.c (hunk 1 above)
Comment (mptre) [inline src/pick.c]: Vim[1]. If your settings is the preferred style of pick lets keep it that way.

[1] https://github.com/mptre/dotfiles/blob/freebsd/vim/ftplugin/c.vim#L1

On Fri, Jan 15, 2016 at 07:28:55AM -0800, Calle Erlandsson wrote:
> > @@ -291,10 +292,10 @@ selected_choice(void)
> >  		case DEL:
> >  			if (cursor_position > 0) {
> >  				delete_between(
> > -				    query,
> > -				    query_length,
> > -				    cursor_position - 1,
> > -				    cursor_position);
> > +					       query,
> > +					       query_length,
> > +					       cursor_position - 1,
> > +					       cursor_position);
> 
> I think these instances were correctly indented. I use `cinoptions=:0,t0,+4,(4` in Vim. What editor and configuration do you use?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/thoughtbot/pick/pull/96/files#r49865276

-- 
:wq

**Comment by calleerlandsson** [comment]
Thanks, @mptre! Merged as 97bb4b151da8dd5f50dd24a8b6969367147f6a09.

**Pull request closed without merging**